### ISSUE TITLE
backbone yolov4

### ISSUE THREAD

**Issue by VeeranjaneyuluToka** [yuto3o/yolox#17]
I have couple of things to understand a bit more.

looks like yolov4 also has darknet53 as backbone with Mish activation, is my understanding correct?
having mish activation and cspdarknet53 backbone improves the accuracy? suppose to improve accuracy based on the paper.

looks like there are different variations of loss functions, would you mind describing scenarios in which a particular loss function recommended or best loss function in ideal cases?

**Comment by yuto3o** [comment]
We follow the cspdarknet53 architecture to build yolov4, and darknet53 block is used to reproduce yolov3 in our repo.

Suffering from the lack of gpu device, the difference or the improvement might not be clear in ablation study.

I think CIoU Loss is a good choice.

**Comment by VeeranjaneyuluToka** [comment]
OK,Thanks for reply!
you have used binary cross entropy to calculate cls_loss, do you notice any benefit using this? I am using categorical cross entropy. And using focal loss would give any benefit?

**Comment by yuto3o** [comment]
Yes, most of the researchers regard the classification task in detection as multi-label learning.
